src/ovp_pipeline/context_binder.py

```
@@ -0,0 +1,679 @@
+:func:`should_rebuild_summary`) live here so BL-084 can plug in
+its own LLM summary writer without re-deriving the policy.  The
+LLM summary call itself is BL-084's responsibility.
+"""
+
+from __future__ import annotations
+
+import logging
+from dataclasses import dataclass
+from datetime import datetime, timezone
+from pathlib import Path
+from typing import Iterable
```

Ruff (0.15.12)

[warning] Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)